### PR TITLE
Add tutorials folder for nni and a link to the Recommenders repo

### PR DESCRIPTION
**Pull request by QuanluZhang**

### CODE REVIEW

File: tutorials/Recommenders_SVD.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+- [Operationalize](https://github.com/Microsoft/Recommenders/blob/master/notebooks/05_operationalize/README.md): Operationalizing models in a production environment on Azure
+
+The fourth task is tuning and optimizing the model's hyperparametrs, this is where NNI could help. To give a concrete example that NNI tunes the models in Recommenders, let's demonstrate with the model [SVD](https://github.com/Microsoft/Recommenders/blob/master/notebooks/02_model/surprise_svd_deep_dive.ipynb), and data Movielens100k. There are more than 10 hyperparameters to be tuned in this model. 
+
+[This Jupyter notebook???]() provided by Recommenders is a very detailed step-by-step tutorial for this example. It uses different built-in tuning algorithms in NNI, including `Annealing`, `SMAC`, `Random Search`, `TPE`, `Hyperband`, `Metis` and `Evolution`. Finally, the results of different tuning algorithms are compared. Please go through this notebook to learn how to use NNI to tune SVD model, then you could further use NNI to tune other models in Recommenders.
```
Comment (xuehui1991) [inline tutorials/Recommenders_SVD.md]: ???

File: tutorials/Recommenders_SVD.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Automatically tuning SVD on NNI
```
Comment (xuehui1991) [inline tutorials/Recommenders_SVD.md]: I am not sure that shall we put this into tutorials? most of the tutorials full of text, but this more like example that full of link.

File: tutorials/Recommenders_SVD.md (hunk 1 above)
Comment (QuanluZhang) [reply tutorials/Recommenders_SVD.md]: will update after their pr merged

File: tutorials/Recommenders_SVD.md (hunk 2 above)
Comment (QuanluZhang) [reply tutorials/Recommenders_SVD.md]: This is indeed a tutorial, the main content is in their repo, I think it is not proper to copy the content directly to our repo. By following the text and links here, users could go through the whole tutorial, which is the goal of this tutorial.

Comment (xuehui1991) [reply tutorials/Recommenders_SVD.md]: I think it more like an example, not a tutorial.

Comment (QuanluZhang) [reply tutorials/Recommenders_SVD.md]: ok, we can move this folder to examples/

File: examples/tutorials/RecommendersSVD.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Automatically tuning SVD on NNI
```
Comment (xuehui1991) [inline examples/tutorials/RecommendersSVD.md]: change the name of tutorials?

Comment (QuanluZhang) [reply examples/tutorials/RecommendersSVD.md]: "practice" is also a good choice, looking for others' opinion.

Comment (PurityFan) [inline examples/tutorials/RecommendersSVD.md]: Suggested to use **RecommendersSvd** as the file name?

**Comment by Crysple** [comment]
PR in the [Recommenders repo](https://github.com/microsoft/recommenders/tree/master/notebooks/04_model_select_and_optimize) was merged.

File: docs/en_US/CommunitySharings/NniPracticeSharing/RecommendersSVD.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Automatically tuning SVD on NNI
+
+In this tutorial, we first introduce a github repo [Recommenders](https://github.com/Microsoft/Recommenders). It is a repository that provides examples and best practices for building recommendation systems, provided as Jupyter notebooks. It has various models that are popular and widely deployed in recommendation systems. To provide a complete end-to-end experience, they present each example in five key tasks, as shown below:
```
Comment (xuehui1991) [inline docs/en_US/CommunitySharings/NniPracticeSharing/RecommendersSVD.md]: I think the folder is too deep......  docs/en_US/CommunitySharings/NniPracticeSharing/RecommendersSVD.md?

File: examples/tutorials/RecommendersSVD.md (hunk 3 above)
Comment (QuanluZhang) [reply examples/tutorials/RecommendersSVD.md]: done

File: docs/en_US/CommunitySharings/NniPracticeSharing/RecommendersSVD.md (hunk 4 above)
Comment (QuanluZhang) [reply docs/en_US/CommunitySharings/NniPracticeSharing/RecommendersSVD.md]: users could find these files through readthedocs.

**Pull request merged**